examples/webgl_animation_scene.html

```diff
@@ -33,156 +33,124 @@
+			var renderer, mixer;
 
 			var clock = new THREE.Clock();
+			var url = 'models/json/scene-animation.json';
 
-			document.addEventListener( 'mousemove', onDocumentMouseMove, false );
+			var SCREEN_WIDTH = window.innerWidth;
+			var SCREEN_HEIGHT = window.innerHeight;
 
-			init();
-			animate();
+			var container = document.getElementById( 'container' );
 
-			function init() {
 
-				container = document.getElementById( 'container' );
+			stats = new Stats();
+			container.appendChild( stats.dom );
 
-				camera = new THREE.PerspectiveCamera( 30, SCREEN_WIDTH / SCREEN_HEIGHT, 1, 10000 );
-				camera.position.z = 150;
+			renderer = new THREE.WebGLRenderer( { antialias: true } );
+			renderer.setClearColor( 0xffffff );
+			renderer.setPixelRatio( window.devicePixelRatio );
+			renderer.setSize( SCREEN_WIDTH, SCREEN_HEIGHT );
+			container.appendChild( renderer.domElement );
 
-				scene = new THREE.Scene();
 
-				scene.fog = new THREE.Fog( 0xffffff, 2000, 10000 );
+			// Load a scene with objects, lights and camera from a JSON file
 
-				//scene.add( camera );
+			new THREE.ObjectLoader().load( url, function ( loadedScene ) {
 
-				// GROUND
+				scene = loadedScene;
 
-				var geometry = new THREE.PlaneBufferGeometry( 16000, 16000 );
-				var material = new THREE.MeshPhongMaterial( { emissive: 0x000000 } );
+				// If the loaded file contains a perspective camera, use it with adjusted aspect ratio...
 
-				var ground = new THREE.Mesh( geometry, material );
-				ground.position.set( 0, FLOOR, 0 );
-				ground.rotation.x = -Math.PI/2;
-				/*scene.add( ground );*/
+				scene.traverse( function ( sceneChild ) {
 
-				ground.receiveShadow = true;
+					if ( sceneChild.type === 'PerspectiveCamera' ) {
 
+						camera = sceneChild;
+						camera.aspect = SCREEN_WIDTH / SCREEN_HEIGHT;
+						camera.updateProjectionMatrix();
 
-				// RENDERER
+					}
 
-				renderer = new THREE.WebGLRenderer( { antialias: true } );
-				renderer.setClearColor( scene.fog.color );
-				renderer.setPixelRatio( window.devicePixelRatio );
-				renderer.setSize( SCREEN_WIDTH, SCREEN_HEIGHT );
-				renderer.domElement.style.position = "relative";
+				} );
 
-				container.appendChild( renderer.domElement );
+				// ... else create a new camera and use it in the loaded scene
 
-				renderer.gammaInput = true;
-				renderer.gammaOutput = true;
+				if ( camera === undefined ) {
 
-				renderer.shadowMap.enabled = true;
+					camera = new THREE.PerspectiveCamera( 30, SCREEN_WIDTH / SCREEN_HEIGHT, 1, 10000 );
+					camera.position.set( - 200, 0, 200 );
 
+				}
 
-				// STATS
+				controls = new THREE.OrbitControls( camera );
 
-				stats = new Stats();
-				container.appendChild( stats.dom );
+				// Ground plane and fog: examples for applying additional children and new property values to the loaded scene
 
-				//
+				var geometry = new THREE.PlaneBufferGeometry( 20000, 20000 );
+				var material = new THREE.MeshStandardMaterial( { roughness: 1 } );
```

Comment:
Of course I would have no problems at all to return to MeshPhongMaterial, since here the intuitivity (as mentioned above) is a minor aspect for me  ;-)
 
But for the sake of interest: Since there are currently even [discussions to replace Phong completely by a new specular/glossy PBR material]( https://github.com/mrdoob/three.js/pull/11038#issuecomment-290964205) I don’t understand: what would be an argument **against** using MeshStandardMaterial in this place? - BTW: Up to now I was not able to notice a significant performance difference (between Phong and Standard) in my test projects. Are there demos that demonstrate a significant difference in relevant scenarios?

> A simpler material (even maybe MeshBasic or MeshLambert) is more appropriate for this use case.

To achieve a similar shading effect like this (which I took from the original example code) with MeshLambertMaterial one would have to tessellate the plane geometry before (factor ca. 200). With MeshBasicMaterial it would not be possible at all.